### PR TITLE
Refine HelpWindow UI

### LINKED ISSUE #155: Refine HelpWindow UI

### PR DESCRIPTION
**Pull request by jereeemyyyy**
# THINGS DONE

* Refine HelpWindow UI
* Added scrollpane

close #155

### CODE REVIEW

File: src/main/resources/view/HelpWindow.css [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+#helpMessage{
+    -fx-text-fill: black;
 }
 
 #copyButton {
-    -fx-background-color: #7D0DC3;
+    -fx-text-fill: white;
+    -fx-font-family: "Arial Rounded MT Bold";
+    -fx-font-weight: bold;
+    -fx-background-color: #a232ea;
```
Comment (ahmadsyuaib) [inline src/main/resources/view/HelpWindow.css:9]: Good changes to UI!

File: src/main/resources/view/HelpWindow.css [hunk 2]
```diff
@@ -15,7 +19,25 @@
...
+.helpMessageWrapper{
+    -fx-background-color: #dedede;
+    -fx-min-height: 5;
+    -fx-min-width: 100;
+    -fx-content-display: CENTER;
+    -fx-alignment: CENTER;
+    -fx-background-radius: 10;
+    -fx-border-radius: 10;
+    -fx-padding: 15;
+
```
Comment (ahmadsyuaib) [inline src/main/resources/view/HelpWindow.css:40]: Nice addition to the UI to fit Client Hub's agreed theme!

File: src/main/resources/view/HelpWindow.fxml [hunk 3]
```diff
@@ -41,92 +49,137 @@
...
+
+            <VBox alignment="CENTER_LEFT" styleClass="commandContainer" VBox.vgrow="NEVER" spacing="5">
+              <Label fx:id="editHelpHeader" alignment="CENTER_LEFT" styleClass="headerLabel" />
+              <HBox styleClass="commandWrapper">
+                <Label fx:id="editHelp" alignment="CENTER_LEFT" styleClass="commandLabel" />
+              </HBox>
+              <padding>
+                <Insets bottom="5" left="10" right="10" top="5" />
+              </padding>
+            </VBox>
```
Comment (ahmadsyuaib) [inline src/main/resources/view/HelpWindow.fxml:88]: Good changes!

**Review by ahmadsyuaib — APPROVED** [review]
Good job Jeremy! Keep it up!

File: src/main/resources/view/HelpWindow.css (hunk 2 above)
Comment (L-rrrr) [reply src/main/resources/view/HelpWindow.css:40]: Good changes to the UI!

File: src/main/resources/view/HelpWindow.fxml (hunk 3 above)
Comment (L-rrrr) [reply src/main/resources/view/HelpWindow.fxml:88]: I like these changes made to the UI! Big improvement from the original addressbook

**Pull request merged**